pkg/host/syscalls_linux.go

```diff
@@ -260,6 +260,13 @@ func isSyzFuseSupported(c *prog.Syscall, target *prog.Target, sandbox string) (b
 	return true, ""
 }
 
+func isSyzUsbIPSupported(c *prog.Syscall, target *prog.Target, sandbox string) (bool, string) {
+	if err := osutil.IsWritable("/sys/devices/platform/vhci_hcd.0/attach"); err != nil {
+		return false, err.Error()
+	}
+	return onlySandboxNone(sandbox)
```

Comment:
Does this require CAP_SYS_ADMIN? Or the file is not accessible with sandbox=namespace? sandbox=namespace should have root access to all files.